lom/src/lib/lomscripted/lom_script.go

```diff
@@ -0,0 +1,424 @@
+package lomscripted
+
+import (
+    "time"
+
+    cmn "lom/src/lib/lomcommon"
+    tele "lom/src/lib/lomtelemetry"
+)
+
+func callGetPubChannel(args []any, cache SuiteCache_t) []any {
```

Comment:
Why do we need this when all APIs are exposed through client.go ?

Reply:
I don't write code to call APIs, but call via data. To enable this, I made all APIs as scriptable.
Now you call *any* API in the same way.

The code you write to call, pass args, validate results -- all declarable in one ScriptEntry_t

Please refer to my test code & tools code.
You don't really see any code - but series of structs 

In short, I don't write code, but talk at a high level via data.

To understand more - please refer to my PPT here - https://microsoft.sharepoint.com/:p:/t/DeviceHealth-DHLocalMitigation/Edskfxiow4BOjLCJn86PLX4BQ_dCnPb3Z5kpQcBjN71VvQ?e=VBJk2k
Look at the sample I have there for OPEN,READ/WRITE & CLOSE file

It is more like, how you would tell someone, say how to publish
1. Get a pub channel 
2. Write your strings
3. close it

This is *now* possible in code too. --- Refer GetPubSuite in **lom/src/tools/toolstelemetry/telemetry_helper.go**
1 == {script.ApiIDGetPubChannel,...}
2 == {script.ApiIDWriteJsonStringsChannel...}
3 == {script.ApiIDCloseChannel